src/tests/issues.targets

```diff
@@ -3699,6 +3699,17 @@
         <ExcludeList Include = "$(XunitTestBinBase)/JIT/Directed/PrimitiveABI/**">
             <Issue>https://github.com/dotnet/runtime/issues/92129</Issue>
         </ExcludeList>
+
+        <!-- TODO: Add tracking issue -->
+        <ExcludeList Include = "$(XunitTestBinBase)/JIT/Directed/coverage/importer/Desktop/**" />
```

Comment:
@jkoritzinsky looks like `XunitTestBinBase` doesn't match `TestRoot` from `ResolveDisabledProjects`. For example, a test is not excluded since the paths differ:
<img width="1031" alt="Screenshot 2025-01-10 at 13 09 02" src="https://github.com/user-attachments/assets/9d75f51f-4fef-439b-a0b6-7fd89087a897" />
<img width="733" alt="Screenshot 2025-01-10 at 12 57 51" src="https://github.com/user-attachments/assets/d5959a91-848b-47b0-b769-d0ae93c3f161" />

Reply:
Yeah the paths differ because the old test harness and the new test harness need to use this exclude list slightly differently due to technical issues. It's one of the reasons I want to move to ActiveIssue attributes and get rid of issues.targets.